CMP1903_A1_2324/Game.cs

```diff
@@ -8,6 +8,10 @@ namespace CMP1903_A1_2324
 {
     internal class Game
     {
+        private Die[] dice;
+
```

Comment:
Style Guidelines state there should be no more than one line of empty space between code.